Documentation/ApiOverview/Logging/Processors/Index.rst

```diff
@@ -100,7 +100,7 @@ WebProcessor
 ------------
 
 The web processor adds selected webserver environment variables to the log record,
-i.e. all possible values from :code:`\TYPO3\CMS\Core\Utility\GeneralUtility::getIndpEnv('_ARRAY')`.
+i.e. all possible values from :code:`\TYPO3\CMS\Core\Http\NormalizedParams`.
```

Comment:
This is unfortunately not true however. Even on the latest master still `GeneralUtility::getIndpEnv('_ARRAY')` is used:

https://github.com/TYPO3/TYPO3.CMS/blob/d7338acaee7776851e14c8d59616e2b7523f88e7/typo3/sysext/core/Classes/Log/Processor/WebProcessor.php#L37

Reply:
urgs.